addons/mail/static/src/components/activity_box/activity_box.scss

```diff
@@ -10,6 +10,11 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ActivityBox_titleLine {
-    border-top: $border-width $border-color dashed;
+.o_ActivityBox_title {
+    //this variable is necessary for the full width chatter margin in mass_mailing
```

Comment:
I would remove the comment, as it feels unnecessary.
If we really want to see where 1st variable is defined, we just grep in code and see it.
(and also comment can become a lie if we change code in mass_mailing!)